config/routes.rb

```diff
@@ -12,6 +12,9 @@
   get '/api/openid_connect/userinfo' => 'openid_connect/user_info#show'
   post '/api/risc/security_events' => 'risc/security_events#create'
   post '/api/irs_attempts_api/security_events' => 'api/irs_attempts_api#create'
+  get '/api/sessions' => 'api/sessions#show'
+  put '/api/sessions' => 'api/sessions#update'
+  delete '/api/sessions' => 'api/sessions#destroy'
```

Comment:
This is an internal API, like only for our own JS. WDYT about prefixing with `/js` or `/api/js` to clarify that it's different from all the endpoints above, which are for external clients? Maybe add a section comment?

Reply:
Interesting thought. We have a lot of other endpoints which are internal in the same way ([examples](https://github.com/18F/identity-idp/blob/fe1d70a73a7b50b0016dff7daa65b63e03ef9bff/config/routes.rb#L32-L34)). Personally, I hadn't considered there being a distinction between internal and public-facing API, at least as reflected in the route paths. I'm not opposed to it, but I'd be curious about naming and how we work to apply the convention consistently. And I suppose we'd probably want the class namespace to reflect it as well.

Maybe something like `/api/internal/sessions` / `Api::Internal::SessionsController` ?

I guess I worry that tying it to JavaScript may be too constraining, though I'm not really sure what other use-case there'd be where we'd expect to be calling these APIs, since the underlying distinction is that these endpoints are stateful and assume to be called with the session cookie.

Another idea could be to flip the expectation, and expect that a public API would be versioned, and an internal API would be any route without a version identifier, or at least a version identifier which marks it as unstable (e.g. `v0`).

Reply:
I pushed a namespacing of 'internal' in 80fd09bc9

Reply:
oh yeah I forgot about those because we alphabetize them lower 🙈 

I like the `/api/internal`! seems like a thing we could consider standardizing on